isis/src/tgo/tsts/uncontrolledSingleColorMosaic/Makefile

```diff
@@ -0,0 +1,55 @@
+# This tests the production of a CaSSIS uncontrolled single-color mosaic from ingestion to
+# export to pds4. 
+#
+# @history 2017-02-13 Kristin Berry - Original version, many things borrowed from 
+#                             singleFrameletProjection cat test. 
+#
+
+INGEST = tgocassis2isis
+SPICE = spiceinit
+MAPFILE = mosrange
+PROJECT = cam2map
+MOSAIC = automos
+EXPORT = isis2pds
+
+include $(ISISROOT)/make/isismake.tsts
+
+
+commands:
+	$(LS) $(INPUT)/*.xml > $(OUTPUT)/inputs.lis;
+	$(SED) 's/^.*input\///g' $(OUTPUT)/inputs.lis > $(OUTPUT)/relative.lis;
+	$(SED) 's/\.xml//g' $(OUTPUT)/relative.lis > $(OUTPUT)/basenames.lis;
+	$(INGEST) $(TSTARGS) from=$(INPUT)/\$$\1.xml \
+            to=$(OUTPUT)/$\$$\1.cub -batchlist=$(OUTPUT)/basenames.lis > /dev/null;
+	$(SPICE) $(TSTARGS) from=$(OUTPUT)/$\$$\1.cub \
+           SPKPREDICTED=true CKPREDICTED=true -batchlist=$(OUTPUT)/basenames.lis > /dev/null;
+	$(LS) $(OUTPUT)/*.cub > $(OUTPUT)/cubes.lis;
+	$(MAPFILE) $(TSTARGS) fromlist=$(OUTPUT)/cubes.lis to=$(OUTPUT)/equi.map > /dev/null;
+	$(PROJECT) $(TSTARGS) from=$(OUTPUT)/$\$$\1.cub to=$(OUTPUT)/$\$$\1_equi.cub \
+             map=$(OUTPUT)/equi.map \
+             pixres=mpp resolution=200 -batchlist=$(OUTPUT)/basenames.lis > /dev/null;
+	$(LS) $(OUTPUT)/*equi.cub > $(OUTPUT)/mosaic.lis;
+	$(MOSAIC) fromlist=$(OUTPUT)/mosaic.lis mosaic=$(OUTPUT)/cassisMosaic.cub > /dev/null;
+	$(EXPORT) from=$(OUTPUT)/cassisMosaic.cub to=$(OUTPUT)/cassisMosaic \
```

Comment:
This line also needs $(TSTARGS) added to it

Reply:
Thanks, @jessemapel!